lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -240,6 +241,10 @@ private function setDriverOptions(array $driverOptions = []) : void
             $supportedDriverOptions[] = MYSQLI_SERVER_PUBLIC_KEY;
         }
 
+        if (defined('MYSQLI_OPT_READ_TIMEOUT')) {
```

Comment:
Why is this condition needed?

Reply:
The option appears to be added in PHP 7.2.

Reply:
The requirement for PHP 7.2 or newer is rock-solid.